index-test.js

```diff
@@ -16,6 +16,82 @@ test("Deleting an entry removes it from the list", () => {
 
 */
 
+setTimeout(() => {
+
+test("adding a new movie title to the list", () => {
+  
+  let movieTitle = document.querySelector(".movie-title")
+  //cheking the id attribute for the button
+  let bingeWorthy = document.querySelector(".movie-name");
+  //to fix the error of inputBtn not being defined
+  const inputBtn = document.querySelector('button')
+  //console.log(ThatGloriusMovie)
+  const creatingTheDiv = document.createElement('div')
+  creatingTheDiv.childElementCount;
+  //console.log(plusTitleToLife)
+   //picks up the input button on click
+   //const inputTitle  = document.querySelector(".list-input")
+  //picks up the input button on click
+  // defining the list input
+  const inputTitle = document.getElementsByClassName('list-input')
+   // giving the input a value otherwise new movie will be created
+  inputTitle.value = "Show me that new movie";
+  // what happens after the click.
+  const expected = inputTitle.childElementCount;
+  //checking the click of an item
+  // then user should then click the button.
+  //checking the click of an item
+  //adding a catcher at the beginning of the function seemed to fix the error of cannot read property of undefined.
+  if(this.item === undefined) {return}
```

Comment:
This was super helpful! Nice to see all your comments too, really clear structure.